tests/test_project.py

```diff
@@ -2037,16 +2036,15 @@ class TestProjectSchema(TestProjectBase):
 
     def test_project_schema_versions(self):
         impossibly_high_schema_version = '9999'
-        assert version.parse(self.project.config['schema_version']) < version.parse(
-            impossibly_high_schema_version)
+        assert version.parse(self.project.config['schema_version']) < \
+            version.parse(impossibly_high_schema_version)
         config = get_config(self.project.fn('signac.rc'))
         config['schema_version'] = impossibly_high_schema_version
         config.write()
         with pytest.raises(IncompatibleSchemaVersion):
-            signac.init_project(
-                name=str(self.project), root=self.project.root_directory())
+            signac.init_project(name=str(self.project), root=self.project.root_directory())
 
-    def test_project_schema_version_migration(self):
+    def test_project_schema_version_mration(self):
```

Comment:
```suggestion
    def test_project_schema_version_migration(self):
```
Typo.

I recommend that you use `git add -up` instead of `git add -u` for a more fine-grained review of the changes that you commit prior to pushing.